app.py

```diff
@@ -15,45 +15,60 @@ def index():
+                updatedStart = MinimalCirclesPath.nextPoint(NotamRequest.startLat, NotamRequest.startLong, bearing, NotamRequest.radius)
+                coordList = MinimalCirclesPath.getPath(updatedStart[0], 
+                                                       updatedStart[1],
+                                                       NotamRequest.destLat,
+                                                       NotamRequest.destLong, 
+                                                       NotamRequest.radius, # circle radius
+                                                       NotamRequest.pathWidth) # path width
+            else:
+                coordList = MinimalCirclesPath.getPath(NotamRequest.startLat, 
+                                                       NotamRequest.startLong,
+                                                       NotamRequest.destLat,
+                                                       NotamRequest.destLong, 
+                                                       NotamRequest.radius, # circle radius
+                                                       NotamRequest.pathWidth) # path width
+            # start timer
+            startTime = time.time() 
 
-        # apiOutputs = [ GetNOTAM.getNotam( NotamRequest.effectiveStartDate,
-        #                                     NotamRequest.effectiveEndDate,
-        #                                     longitude, # longitude
-        #                                     latitude, # latitude
-        #                                     1, # page num
-        #                                     NotamRequest.radius) #page num here is one temporarily
-        #                                     for latitude, longitude in coordList ]
-        apiOutputs = []
+            # call the API for each point
+            print("LOADING...")
 
-        for latitude, longitude in coordList:
-            new_data = GetNOTAM.buildNotam(NotamRequest.effectiveStartDate, NotamRequest.effectiveEndDate, longitude, latitude, NotamRequest.radius)
-            apiOutputs.extend(new_data)
+            # apiOutputs = [ GetNOTAM.getNotam( NotamRequest.effectiveStartDate,
+            #                                     NotamRequest.effectiveEndDate,
+            #                                     longitude, # longitude
+            #                                     latitude, # latitude
+            #                                     1, # page num
+            #                                     NotamRequest.radius) #page num here is one temporarily
+            #                                     for latitude, longitude in coordList ]
+            
 
-        for latitude, longitude in coordList:
-            new_data = GetNOTAM.buildNotam(NotamRequest.effectiveStartDate, NotamRequest.effectiveEndDate, longitude, latitude, NotamRequest.radius)
-            apiOutputs.extend(new_data)
-        
+            for latitude, longitude in coordList:

```

Comment:
id recommend this loop and some of the other lines need atleast brief comments describing what their purpose it. for example the if statement above # if ____ do this other wise _____